css/store.css

```diff
@@ -0,0 +1,171 @@
+/* ---------- Header style ---------- */
+
+nav li:nth-child(3) a {
+  color: var(--dark-orange);
+}
+
+header {
+  border-bottom: lightgrey solid 1px;
+}
+
+/* ---------- Main style  ---------- */
+
+main {
+  display: flex;
+  height: 95vh;
+  background-color: white;
+}
+
+/* ---------- Main style (SECTION LEFT) ---------- */
+
+.left-section {
+  width: 50%;
+  padding: 50px 5vw 30px 5vw;
+}
+
+.left-section h1 {
+  color: var(--dark-orange);
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+fieldset {
+  border: none;
+  padding: 0;
+  margin: 0;
+}
+
+fieldset div {
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+label {
+  color: var(--dark-grey);
+  font-size: 0.8rem;
+  font-weight: 500;
+  margin-bottom: 0.5em;
+}
+
+input {
+  border: 1px solid var(--dark-grey);
+  border-radius: 3px;
+  padding: 7px;
+  width: 100%;
+}
+
+/* ---------- Name and address style (SECTION LEFT) ---------- */
+
+.name-fieldset {
+  width: 100%;
+  display: flex;
+  gap: 15px;
+  justify-content: space-between;
+}
+
+.name-fieldset div {
+  width: 50%;
+}
+
+.city-postcode-div {
+  width: 100%;
+  gap: 15px;
+  flex-direction: row;
+}
+
+.city-div {
```

Comment:
I would not add `div` to the class name. It's just an HTML element, and not needed to describe the element you want to apply styles to.